src/components/editor/metaform-editor-left-drawer.tsx

```diff
@@ -25,15 +26,21 @@ const MetaformEditorLeftDrawer: React.FC<Props> = ({
   setPendingForm
 }) => {
   const [ tabIndex, setTabIndex ] = React.useState(0);
+  const currentHostname = window.location.hostname;
 
   /**
    * Event handler for metaform property change
    *
    * @param key metaform property key
+   * @param key2 metaform slug
    */
-  const onMetaformPropertyChange = (key: keyof Metaform): ChangeEventHandler<HTMLInputElement> =>
+  const onMetaformPropertyChange = (key: keyof Metaform, key2: keyof Metaform): ChangeEventHandler<HTMLInputElement> =>
     ({ target: { value } }) => {
-      pendingForm && setPendingForm({ ...pendingForm, [key]: value });
+      pendingForm && setPendingForm({
+        ...pendingForm,
+        [key]: value,
+        [key2]: slugify(value, { lower: true })
```

Comment:
If this method is only used for changing title and slug, then the naming could be more descriptive. Change method name and keys to title & slug